### DISCUSSION TITLE
Offboard Wizard

### DISCUSSION THREAD

**Discussion by steven-techtron** [KelvinTegelaar/CIPP#786] [Ideas]
Would be nice when finished running the Offboard Wizard their was a button to run the wizzard again, for the scenario where you are offboarding more than one user.  If you click the offboard menu item once you have finished the process it just stays on the summary screen.  You need to select another menu option and then select offboard wizard again to start at step one.

Thanks for the amazing work!

**Comment by smartproit** [comment]
Or an option to select multiple users and perform the offboard on all of them at once.

Even without this it's a huge time saver and we're very appreciative. Keep up the good work!
